lib/instance-validator.js

```diff
@@ -320,17 +321,20 @@ class InstanceValidator {
    */
   _validateSchema(rawAttribute, field, value) {
     if (rawAttribute.allowNull === false && (value === null || value === undefined)) {
-      const validators = this.modelInstance.validators[field];
-      const errMsg = _.get(validators, 'notNull.msg', `${this.modelInstance.constructor.name}.${field} cannot be null`);
-
-      this.errors.push(new sequelizeError.ValidationErrorItem(
-        errMsg,
-        'notNull Violation', // sequelizeError.ValidationErrorItem.Origins.CORE,
-        field,
-        value,
-        this.modelInstance,
-        'is_null'
-      ));
+      const association = _.values(rawAttribute.Model.associations).find(association => association instanceof BelongsTo && association.foreignKey === rawAttribute.fieldName);
```

Comment:
`rawAttribute.Model.associations`  => `this.modelInstance.constructor.associations`, just this last change :)